### PR TITLE
Feature/fixing borders offset

### PR DESCRIPTION
**Pull request by andrey-abramow**

### CODE REVIEW

File: build/angular-range-slider.js [hunk 1]
```diff
@@ -34,12 +36,13 @@
...
-            return rightBubblePosition = -minElement.offsetWidth;
+            bubbleSize = minElement.clientWidth;
+            minElement.style.left = -bubbleErrorOffset + 'px';
+            return rightBubblePosition = -bubbleErrorOffset;
           });
           scope.$watch(maxElement, function() {
-            maxElement.style.right = -maxElement.offsetWidth + 'px';
-            return leftBubblePosition = -minElement.offsetWidth;
+            maxElement.style.right = -bubbleErrorOffset + 'px';
+            return leftBubblePosition = -bubbleErrorOffset;
```
Comment (Rademade-review) [inline build/angular-range-slider.js]: Did you mean to return a conditional instead of an assignment?

File: build/angular-range-slider.js [hunk 2]
```diff
@@ -34,12 +36,13 @@
           rightBubblePosition = 0;
           leftBubblePosition = 0;
           scope.$watch(minElement, function() {
-            minElement.style.left = -minElement.offsetWidth + 'px';
-            return rightBubblePosition = -minElement.offsetWidth;
+            bubbleSize = minElement.clientWidth;
+            minElement.style.left = -bubbleErrorOffset + 'px';
+            return rightBubblePosition = -bubbleErrorOffset;
```
Comment (Rademade-review) [inline build/angular-range-slider.js]: Did you mean to return a conditional instead of an assignment?

File: build/angular-range-slider.js [hunk 3]
```diff
@@ -12,12 +12,14 @@
...
         link: function(scope) {
-          var MAX_BUBBLE, MIN_BUBBLE, calculatePosition, currentDragBubble, dragBubble, dropBubble, finishPosition, getPixelsOfSliderRangeProperty, initMaxValue, initMinValue, leftBubblePosition, maxElement, maxPosition, maxWidthRange, minElement, minPosition, moveBubble, resetPosition, rightBubblePosition, setSliderLeftPosition, setSliderPosition, setSliderRightPosition, sliderContainer, sliderRange, sliderRangeCurrentX, startPosition, step, updateValues, _initialize;
+          var MAX_BUBBLE, MIN_BUBBLE, bubbleErrorOffset, bubbleSize, calculatePosition, currentDragBubble, dragBubble, dropBubble, finishPosition, getPixelsOfSliderRangeProperty, initMaxValue, initMinValue, leftBubblePosition, maxElement, maxPosition, maxWidthRange, minElement, minPosition, moveBubble, resetPosition, rightBubblePosition, setSliderLeftPosition, setSliderPosition, setSliderRightPosition, sliderContainer, sliderRange, sliderRangeCurrentX, startPosition, step, updateValues, _initialize;
```
Comment (Rademade-review) [inline build/angular-range-slider.js]: Line is too long.

File: src/js/slider/slider.coffee [hunk 4]
```diff
@@ -25,7 +25,9 @@ angular.module('ngSlider',[]).directive 'slider',[ ->
     maxElement = document.getElementsByClassName('slider-btn max')[0]
     sliderContainer = document.getElementsByClassName('slider-container')[0]
     sliderRange = document.getElementById('slider-range')
-    maxWidthRange = sliderContainer.clientWidth
+    bubbleSize = minElement.clientWidth
+    bubbleErrorOffset = 1
```
Comment (Melanitski) [inline src/js/slider/slider.coffee]: Почему именно errorOffset? 
Можем мы эту переменную сделать настраиваемую?

File: build/angular-range-slider.js [hunk 5]
```diff
@@ -34,12 +37,13 @@
...
-            return rightBubblePosition = -minElement.offsetWidth;
+            bubbleSize = minElement.clientWidth;
+            minElement.style.left = -bubbleOffset + 'px';
+            return rightBubblePosition = -bubbleOffset;
           });
           scope.$watch(maxElement, function() {
-            maxElement.style.right = -maxElement.offsetWidth + 'px';
-            return leftBubblePosition = -minElement.offsetWidth;
+            maxElement.style.right = -bubbleOffset + 'px';
+            return leftBubblePosition = -bubbleOffset;
```
Comment (Rademade-review) [inline build/angular-range-slider.js:46]: Did you mean to return a conditional instead of an assignment?

File: build/angular-range-slider.js [hunk 6]
```diff
@@ -8,16 +8,19 @@
...
+          var MAX_BUBBLE, MIN_BUBBLE, bubbleOffset, bubbleSize, calculatePosition, currentDragBubble, dragBubble, dropBubble, finishPosition, getPixelsOfSliderRangeProperty, initMaxValue, initMinValue, leftBubblePosition, maxElement, maxPosition, maxWidthRange, minElement, minPosition, moveBubble, resetPosition, rightBubblePosition, setSliderLeftPosition, setSliderPosition, setSliderRightPosition, sliderContainer, sliderRange, sliderRangeCurrentX, startPosition, step, updateValues, _initialize;
           minElement = document.getElementById('slider-btn-min');
           maxElement = document.getElementsByClassName('slider-btn max')[0];
           sliderContainer = document.getElementsByClassName('slider-container')[0];
           sliderRange = document.getElementById('slider-range');
-          maxWidthRange = sliderContainer.clientWidth;
+          bubbleSize = minElement.clientWidth;
+          bubbleOffset = scope.bubbleOffset | 1;
```
Comment (Rademade-review) [inline build/angular-range-slider.js:22]: Unexpected use of '|'.

File: build/angular-range-slider.js [hunk 7]
```diff
@@ -8,16 +8,19 @@
...
         link: function(scope) {
-          var MAX_BUBBLE, MIN_BUBBLE, calculatePosition, currentDragBubble, dragBubble, dropBubble, finishPosition, getPixelsOfSliderRangeProperty, initMaxValue, initMinValue, leftBubblePosition, maxElement, maxPosition, maxWidthRange, minElement, minPosition, moveBubble, resetPosition, rightBubblePosition, setSliderLeftPosition, setSliderPosition, setSliderRightPosition, sliderContainer, sliderRange, sliderRangeCurrentX, startPosition, step, updateValues, _initialize;
+          var MAX_BUBBLE, MIN_BUBBLE, bubbleOffset, bubbleSize, calculatePosition, currentDragBubble, dragBubble, dropBubble, finishPosition, getPixelsOfSliderRangeProperty, initMaxValue, initMinValue, leftBubblePosition, maxElement, maxPosition, maxWidthRange, minElement, minPosition, moveBubble, resetPosition, rightBubblePosition, setSliderLeftPosition, setSliderPosition, setSliderRightPosition, sliderContainer, sliderRange, sliderRangeCurrentX, startPosition, step, updateValues, _initialize;
```
Comment (Rademade-review) [inline build/angular-range-slider.js:16]: Line is too long.

File: src/js/slider/slider.coffee (hunk 4 above)
Comment (andrey-abramow) [inline src/js/slider/slider.coffee]: добавил

**Pull request merged**